docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+
+services:
+    postgres:
+      image: postgres
+      environment: 
+        POSTGRES_PASSWORD: postgres
+        POSTGRES_DB: bit_schema
+      #options: 
+        #--health-cmd pg_isready
+        #--health-interval 10s
+        #--health-timeout 5s
+        #--health-retries 5
+      ports:
+        - 5432:5432
+      networks: 
+        - custom_network
+
+    bit:
+      container_name: BIT 
+      build: . 
+      ports: 
+        - 5000:5000
+      networks:
+        - custom_network
+
+    ms: 
```

Comment:
Here the MS is using port 4000

Reply:
Also, how do we run the MS backend application in this scenario? you're referring here as custom network MS:4000? Where is the code base where this MS server is running from? Does this mean we also need to run from docker on MS backend server for BIT? We should open an issue for this docker on the MS backend `bit` branch, right? 
I'm new to docker, so would like to learn more 😉

Reply:
Nope the contributors would still need to clone the MS backend however it would be as simple as `git clone.....` and then when they run this file it would automatically setup the MS backend as well.

Reply:
> Also, how do we run the MS backend application in this scenario? you're referring here as custom network MS:4000? Where is the code base where this MS server is running from? Does this mean we also need to run from docker on MS backend server for BIT? We should open an issue for this docker on the MS backend `bit` branch, right?
> I'm new to docker, so would like to learn more 😉

The custom network is something that is a part of docker but it nope it doesn't mean the MS:4000 I am not even entirely sure we need it anymore I will remove it. We can just run the containers on the default network.

Reply:
@mtreacy002 I just found a way to make docker run the program without the contributor having to clone the MS backend. Would you like me to implement that as well? For reference an [SO answer](https://stackoverflow.com/questions/26753030/how-to-build-docker-image-from-github-repository/39194765)